contents.json

```diff
@@ -5766,6 +5766,11 @@
       "category": "utility",
       "description": "📦 KeyPath dynamicMemberLookup based syntax sugar for Swift.",
       "homepage": "https://github.com/marty-suzuki/DuctTape"
+    }, {
+      "title": "SwiftFormat",
+      "category": "quality",
+      "description": "A code library and command-line formatting tool for reformatting Swift code",
```

Comment:
Hi @IceCreamCremo, could you add a full stop/period to the end of description?

Reply:
done